engines/sci/graphics/animate.cpp

```diff
@@ -650,6 +650,11 @@ void GfxAnimate::animateShowPic() {
 }
 
 void GfxAnimate::kernelAnimate(reg_t listReference, bool cycle, int argc, reg_t *argv) {
+	// If necessary, delay this kAnimate for a running PalVary.
+	// See delayForPalVaryWorkaround() for details.
+	if (_screen->_picNotValid)
```

Comment:
Is this going to be a problem in SCI1.1 which also has a `_picNotValidSci11`?

Reply:
This check is the same check that is used for calling `animateShowPic()` later in this function, so this should be the right one.